src/Storefront/Controller/VerificationHashController.php

```diff
@@ -0,0 +1,48 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Storefront\Controller;
+
+use Shopware\Core\Framework\Routing\Annotation\RouteScope;
+use Shopware\Core\System\SalesChannel\SalesChannelContext;
+use Shopware\Core\System\SystemConfig\SystemConfigService;
+use Shopware\Storefront\Exception\VerificationHashNotConfiguredException;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+/**
+ * @RouteScope(scopes={"storefront"})
```

Comment:
This assumes that the Store License domain must be an sales channel. The licenseHost can currently be anything.
https://github.com/shopware/platform/pull/572/files#diff-bfa3c20d9d35a22675e5b7e5adc13bdfR9

So the controller is not accessible then

Reply:
Which scope do you suggest instead? IIRC there is an exception when no scope is given.

Reply:
We should change it to the api scope, because it's part of the core. 
```suggestion
 * @RouteScope(scopes={"api"})
```

Reply:
When I change the scope I have to add the attribute that I don't need credentials for authentication, right?

Reply:
You're too fast :) See my newest comment

Reply:
You're too slow! 
![grafik](https://user-images.githubusercontent.com/1133593/90012716-38911180-dca4-11ea-918b-7c59ebc67f9f.png)
I'll wait next time :D I see where this is going